src/Mod/PartDesign/Gui/TaskFeaturePick.cpp

```diff
@@ -58,6 +58,7 @@
 using namespace PartDesignGui;
 using namespace Attacher;
 
+// ***WMAYER, this comment needs revising***
 // TODO Do ve should snap here to App:Part or GeoFeatureGroup/DocumentObjectGroup ? (2015-09-04, Fat-Zer)
```

Comment:
What should this comment say? (I'll remove line 61 once you answer)

Reply:
leave it there for now, there are a lot of those in the source code, no way to solve everything at once

Reply:
@yorikvanhavre I was referring to `TODO Do ve should snap` what does `Do ve` mean?

Reply:
not sure... Better to leave it like that

Reply:
@wmayer any idea what Fat-Zer was trying to communicate ?

Reply:
@yorikvanhavre the comment that I added needed to be wiped before merge :man_facepalming: